src/data_structs/layout.jl

```diff
@@ -87,6 +87,9 @@ function Base.getproperty(obj::MultiBubble, sym::Symbol)
     end
 end  # function Base.getproperty
 
+Rfuse(cs::SingleBubble) = cs.radius
+Rfuse(cs::MultiBubble) = cs.radius
```

Comment:
Not quite sure why we need this but even if we do we don't need to go define this for every type. If the structure has a radius it will return a radius
```suggestion
Rfuse(cs) = cs.radius 
```